docs/metrics.md

```diff
@@ -139,6 +204,36 @@ The logs will be printed to stdout in the standard [mozlog format](https://githu
 }
 ```
 
+And Redshift schema:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
+    {"uuid",                       "VARCHAR",   36,      nil,         get_uuid},
+    {"Hostname",                   "VARCHAR",   255,     nil,         "Hostname"},
+    {"logger",                     "VARCHAR",   255,     nil,         "logger"},
+    {"Severity",                   "INTEGER",   nil,     nil,         "Severity"},
+    {"type",                       "VARCHAR",   255,     nil,         "type"},
+
+    {"agent",                      "VARCHAR",   255,     nil,         "Fields[agent]"},
+    {"errno",                      "VARCHAR",   255,     nil,         "Fields[errno]"},
+    {"lang",                       "VARCHAR",   255,     nil,         "Fields[lang]"},
+    {"method",                     "VARCHAR",   255,     nil,         "Fields[method]"},
+    {"path",                       "VARCHAR",   255,     nil,         "Fields[path]"},
+    {"status_code",                "INTEGER",   nil,     nil,         "Fields[status_code]"},
+    {"t",                          "VARCHAR",   36,      nil,         "Fields[t]"},
+
+    {"classifiers",                "VARCHAR",   1000,    nil,         "Fields[classifiers]"},
+
+    {"predicate__is_protocol",     "BOOLEAN",   nil,     nil,         "Fields[predicate.is_protocol]"},
```

Comment:
This is some funky double-underscoring. There are lowerCamelCased keys in the client data. Let's pick one, I don't care which, and be consistent throughout. Is one preferred for redshift?